Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/ComposerSkillBotDotNet.dialog

```diff
@@ -0,0 +1,187 @@
+                  },
+                  "activityProcessed": true,
+                  "dialog": "AuthDialog",
+                  "options": {}
+                }
+              ]
+            },
+            {
+              "value": "MessageWithAttachment",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "as1hYq"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "MessageWithAttachmentDialog"
+                }
+              ]
+            },
+            {
+              "value": "Sso",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "tQ6oU7"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "SsoSkillDialog"
+                }
+              ]
+            },
+            {
+              "value": "FileUpload",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "eFvfkP"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "FileUploadDialog"
+                }
+              ]
+            },
+            {
+              "value": "Echo",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "CmjjTc"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "CallEchoSkill",
+                  "options": {}
+                }
+              ]
+            },
+            {
+              "value": "Delete",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "xxKIQ2"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "DeleteDialog"
+                }
+              ]
+            },
+            {
+              "value": "Update",
+              "actions": [
+                {
+                  "$kind": "Microsoft.BeginDialog",
+                  "$designer": {
+                    "id": "oLQRy6"
+                  },
+                  "activityProcessed": true,
+                  "dialog": "UpdateDialog"
+                }
+              ]
+            }
+          ],
+          "default": [
+            {
+              "$kind": "Microsoft.SendActivity",
+              "$designer": {
+                "id": "Wz6ESA"
+              },
+              "activity": "${SendActivity_Wz6ESA()}"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "generator": "ComposerSkillBotDotNet.lg",
+  "id": "ComposerSkillBotDotNet",
+  "recognizer": "ComposerSkillBotDotNet.lu.qna"
```

Comment:
Change the default recognizer to Regular Expressions so composer doesn't try to run luid build and other LUIS related checks

![image](https://user-images.githubusercontent.com/12264946/144068040-ace992e7-9e09-4a62-93d5-2b630f0a1195.png)

Reply:
Done. Thanks!

Reply:
Done, thanks!